recipes/upa-url/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import copy, get, rmdir
+import os
+
+required_conan_version = ">=2"
+
+class upa_urlRecipe(ConanFile):
+    name = "upa-url"
+
+    # Optional metadata
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/upa-url/upa"
+    description = "An implementation of the WHATWG URL Standard in C++"
+    topics = ("url", "parser", "psl", "whatwg")
+
+    # Binary configuration
+    package_type = "static-library"
```

Comment:
This is a static library for now until the next release, see https://github.com/upa-url/upa/pull/124